packages/beacon-node/test/unit/api/impl/events/events.test.ts

```diff
@@ -24,6 +25,7 @@ describe("Events api impl", function () {
       chainStub.stateCache = (stateCacheStub as unknown) as StateContextCache;
       chainEventEmmitter = new ChainEventEmitter();
       chainStub.emitter = chainEventEmmitter;
+      (chainStub as {forkChoice: IForkChoice}).forkChoice = ({getBlock: sinon.stub()} as unknown) as IForkChoice;
```

Comment:
This stub is not necessary anymore right?

Reply:
no it's not necessary anymore, I removed it 👍